### ISSUE TITLE
[Feature Request] Get only date or only time

### ISSUE THREAD

**Issue by JCKodel** [jogboms/time.dart#23]
In C#, we have those two usefull properties of a DateTime:

```csharp
var now = DateTime.Now; // 2020-04-10T15:27:00
var datePartOnly = now.Date; // 2020-03-10T00:00:00 (returns a DateTime, with 00:00:00 time)
var timePartOnly = now.TimeOfDay; // 15:17:00 (returns a TimeSpan - Duration in Dart)
```

**Comment by shinriyo** [comment]
@JCKodel
Ah, you mean you want only date and time properties?

**Comment by shinriyo** [comment]
Is `2020-03-10T00:00:00` `2020-04-10T00:00:00`?
Is `15:17:00` `15:27:00`?

**Comment by shinriyo** [comment]
@JCKodel I add properties. Is it okay with you?

> returns a TimeSpan - Duration

The `timePartOnly` I added returns a string.
Dart doesn't have TimeSpan.

**Comment by jogboms** [comment]
Hey @shinriyo 
Nice work with the PR. I would say, instead of a string from `timeOfDay`, maybe a `Duration` as a replacement for `TimeSpan`.

**Comment by jogboms** [comment]
On second thought, is there really any added benefits in transforming time into a duration?

**Comment by JCKodel** [comment]
> On second thought, is there really any added benefits in transforming time into a duration?

Yes yes yes!

I'm using this to calculate 12 events with a base date (date of the first event), then adding two days for each event. Every event has an alarm time (so it is easier to calc `currentDate + Duration`)

**Comment by jogboms** [comment]
Oh yes, you could definitely do that `currentDate + 2.days` is also valid.

**Comment by jogboms** [comment]
Closed by #25
